src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -17,6 +17,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
+using nuint = System.UInt64;
```

Comment:
I don't understand why we aren't just using `ulong` here. ??

Reply:
Realistically, we should `#ifdef` this and use `System.UInt32` on 32-bit systems and `System.UInt64` on 64-bit systems (since C# still doesn't expose the appropriate operators for `System.IntPtr` and `System.UIntPtr`).

Reply:
We don't have the capability of building architecture-specific IL today.

Reply:
Also, what does `nuint` stand for?